packages/n4s/src/enforce/compounds/shape.js

```diff
@@ -23,11 +25,15 @@ export default function shape(obj, shapeObj) {
     }
   }
 
-  for (const key in obj) {
-    if (!shapeObj[key]) {
-      return false;
+  if (!(options || {}).loose) {
```

Comment:
I would usually opt for a default parameter (assuming options is either an object or undefined), but for now - since this is only limited to this line, and the transpiled size _is_ smaller with your suggested style, I'll take it 👍 .